src/neo/Network/P2P/MessageCommand.cs

```diff
@@ -122,6 +122,12 @@ public enum MessageCommand : byte
         [ReflectionCache(typeof(Block))]
         Block = 0x2c,
 
+        /// <summary>
+        /// Sent to send an <see cref="NotaryRequest"/>.
+        /// </summary>
+        [ReflectionCache(typeof(NotaryRequest))]
+        Notary = 0x50,
```

Comment:
Use `Extensible`?

Reply:
Not everyone can send extansible paylaod. Users need this request to send their signature so that node with notary plugin can collect them.

Reply:
> Use `Extensible` payload?

Maybe difficult. Because `RequestPayload `has a more complex verification logic, `extansible paylaod` is not enough to support
`        public bool Verify(ProtocolSettings settings)
        {
            var nKeysMain = MainTransaction.GetAttributes<NotaryAssisted>();
            if (!nKeysMain.Any()) return false;
            if (nKeysMain.ToArray()[0].NKeys == 0) return false;
            if (!fallbackTransaction.Script.SequenceEqual(FallbackFixedScript)) return false;
            if (FallbackTransaction.Signers.Length != 2) return false;
            if (fallbackTransaction.Signers[1].Scopes != WitnessScope.None) return false;
            if (FallbackTransaction.Witnesses[0].InvocationScript.Length != 66
                || FallbackTransaction.Witnesses[0].VerificationScript.Length != 0
                || (FallbackTransaction.Witnesses[0].InvocationScript[0] != (byte)OpCode.PUSHDATA1 && FallbackTransaction.Witnesses[0].InvocationScript[1] != 64))
                return false;
            if (FallbackTransaction.GetAttribute<NotValidBefore>() is null) return false;
            var conflicts = FallbackTransaction.GetAttributes<ConflictAttribute>();
            if (conflicts.Count() != 1) return false;
            if (conflicts.ToArray()[0].Hash != MainTransaction.Hash) return false;
            var nKeysFallback = FallbackTransaction.GetAttributes<NotaryAssisted>();
            if (!nKeysFallback.Any()) return false;
            if (nKeysFallback.ToArray()[0].NKeys != 0) return false;
            if (MainTransaction.ValidUntilBlock != FallbackTransaction.ValidUntilBlock) return false;
            if (!fallbackTransaction.VerifyWitness(settings, null, fallbackTransaction.Signers[1].Account, fallbackTransaction.Witnesses[1], 0_02000000, out _)) return false;
            return this.VerifyWitnesses(settings, null, 0_02000000);
        }`

Reply:
though i saw this before, but still, this logic is increadblly complex~~~